tests/fixtures/small_invalid_file

```diff
@@ -0,0 +1,23 @@
+This file is valid for the upload-service functional tests because it contains an odd number of bytes.
```

Comment:
Do you mean to say the file is invalid here?

Reply:
Oops. Yep, done.